cmd/genz/genz.go

```diff
@@ -67,9 +69,23 @@ func (c generateCommand) Run() error {
 		tags = strings.Split(*buildTags, ",")
 	}
 
-	template, err := os.ReadFile(*templateFile)
-	if err != nil {
-		return fmt.Errorf("failed to read template file %s: %v", *templateFile, err)
+	var template []byte
+	if strings.HasPrefix(*templateFile, "http://") || strings.HasPrefix(*templateFile, "https://") {
+		response, err := http.Get(*templateFile)
+		if err != nil {
+			return fmt.Errorf("failed to make a request to %s: %v", *templateFile, err)
+		}
+		body, err := ioutil.ReadAll(response.Body)
```

Comment:
Use `io` instead of `ioutil` as it has been deprecated since 1.16 . See: https://pkg.go.dev/io/ioutil